Remove the hard systemd_dep, only required for getting the systemduserunitdir
--- a/meson.build.orig	2020-08-31 12:05:57.621500291 +0200
+++ b/meson.build	2020-08-31 12:06:27.997894754 +0200
@@ -130,9 +130,7 @@
 
   # Check for systemd
   if enable_systemd
-    systemd_dep = dependency('systemd', version: '>= 242', required: true)
-    systemd_userunitdir = systemd_dep.get_pkgconfig_variable('systemduserunitdir',
-                                                             define_variable: ['prefix', prefix])
+    systemd_userunitdir = '/DELETEME'
 
     libsystemd_dep = dependency('libsystemd', version: '>= 209', required: false)
     session_bin_deps += libsystemd_dep
